typescript/infra/scripts/funding/utils/utils.ts

```diff
@@ -0,0 +1,5 @@
+export function formatDailyBurn(dailyBurn: number) {
```

Comment:
this feels like a handy to thing to have in utils? like more generically "format to X sf"